cmd/manager/cmd/root.go

```diff
@@ -41,7 +41,7 @@ var (
 var rootCmd = &cobra.Command{
 	Use:   "manager",
 	Short: "The central manager of dragonfly.",
-	Long: `manager is a long-running process and is mainly responsible 
+	Long: `manager is a long-running process and is mainly responsible
```

Comment:
This is because my editor is triming trailing whitespace automatically, I think this change is ok, since there is no difference for human when reading the help output(it wraps line here).